mod/ping.php

```diff
@@ -201,6 +201,7 @@ function ping_init(App $a)
 		if (is_null($ev)) {
 			$ev = qu("SELECT count(`event`.`id`) AS total, type, start, adjust FROM `event`
 				WHERE `event`.`uid` = %d AND `start` < '%s' AND `finish` > '%s' and `ignore` = 0
+				GROUP BY type, start, adjust
```

Comment:
As suggested by line 219-220 below, `$ev` is supposed to be an array of the events in the next 7 days. The `COUNT` seems to be superfluous here, as it can be obtained by doing `count($ev)`.